### PR TITLE
fix(search-indexes): smaller column widths COMPASS-7341

### PR DESCRIPTION
**Pull request by mabaasit**
In this PR, added following UX changes:
1. Smaller table column widths in Search Indexes
2. Align Index Actions to the right

### Checklist
- [ ] New tests and/or benchmarks are included
- [ ] Documentation is changed or added
- [ ] I have signed the MongoDB Contributor License Agreement (https://www.mongodb.com/legal/contributor-agreement)

## Motivation and Context

- [ ] Bugfix
- [ ] New feature
- [ ] Dependency update
- [ ] Misc

## Types of changes

- [ ] *Backport Needed*
- [ ] Patch (non-breaking change which fixes an issue)
- [ ] Minor (non-breaking change which adds functionality)
- [ ] Major (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -58298,7 +58298,7 @@
         "@leafygreen-ui/hooks": "^7.3.3",
         "@leafygreen-ui/icon": "^11.21.0",
         "@leafygreen-ui/icon-button": "^15.0.3",
-        "@leafygreen-ui/info-sprinkle": "*",
+        "@leafygreen-ui/info-sprinkle": "^1.0.0",
```
Comment (kmruiz) [inline package-lock.json:58301]: Is this change on purpose?

Comment (mabaasit) [reply package-lock.json:58301]: `npm run bootstrap` made this change. as CI passes, I think its fine
